x-pack/solutions/observability/plugins/synthetics/server/alert_rules/status_rule/queries/query_monitor_status_alert.ts

```diff
@@ -164,6 +263,16 @@ export async function queryMonitorStatusAlert({
             const configId = latestPing.config_id;
             const monitorQueryId = latestPing.monitor.id;
 
+            const msSinceLastPing =
+              new Date().getTime() - new Date(latestPing['@timestamp']).getTime();
+            const msBeforeIsPending =
+              monitorsData[monitorQueryId].scheduleInMs +
+              moment.duration(60, 'seconds').asMilliseconds();
+
+            // Example: if a monitor has a schedule of 5m the last valid ping can be at (5+1)m
+            // If it's greater than that it means the monitor is pending
+            const isValidPing = msBeforeIsPending - msSinceLastPing > 0;
+
```

Comment:
probably create a simple function and write unit tests for simple logic like this

Reply:
Done